converter/src/test/resources/pcf/acceptance2021/success/PCF-11d-CMS122v9UpperCaseUUID_06292021.xml

```diff
@@ -672,8 +672,8 @@ QRDA Category III Measure Section (CMS EP)
                            codeSystem="2.16.840.1.113883.6.96"
                            displayName="Observation Parameters"/>
                      <effectiveTime>
-                        <low value="20210101"/>
-                        <high value="20211231"/>
+                        <low value="20220101"/>
+                        <high value="20221231"/>
```

Comment:
These are still in the directory `acceptance2021` - is that right?

Reply:
Yes, there will be another ticket for replacing these with 2022 files. For now, since Conversion Tool is still in an in between Y5 and Y6, we will use this.